source/module_base/test/realarray_test.cpp

```diff
@@ -199,3 +201,13 @@ TEST_F(realArrayTest,Alloc)
 	output = testing::internal::GetCapturedStdout();
 	EXPECT_THAT(output,testing::HasSubstr("Allocation error for realArray"));
 }
+
+TEST_F(realArrayTest,Zeros)
+{
+        int a[100];
```

Comment:
Please add another example of double array.

Reply:
Please use a pointer to initialize a double array and then call zeros. The prototype of the target function is:
```
void ModuleBase::zeros<double>(double*, int)
```